app/assets/javascripts/dropzone.js.es6

```diff
@@ -0,0 +1,66 @@
+var file_input_query = document.querySelectorAll("input[type='file']")
+if (file_input_query.length == 1) {
+    insert_dropzone(file_input_query[0])
+}
+  
+function insert_dropzone(file_input) {
+    //create and insert dropzone
```

Comment:
Can you restructure this file as a class with methods instead of bear a method? Similar in structure to `app/assets/javascripts/table_search.js.es6`.  It should have a constructor and method for enabling the event listeners.

The last line in the file can then call new on itself so that it executes automatically.